proto/proto/penumbra/core/dex/v1alpha1/dex.proto

```diff
@@ -27,43 +27,46 @@ message SwapClaim {
   bytes proof = 1;
   // Encapsulates the authorized fields of the SwapClaim action, used in signing.
   SwapClaimBody body = 2;
+  // The epoch duration of the chain when the swap claim took place.
+  uint64 epoch_duration = 7;
 }
 
 // Encapsulates the authorized fields of the SwapClaim action, used in signing.
 message SwapClaimBody {
-  // The nullifier for the Swap NFT to be consumed.
+  // The nullifier for the Swap commitment to be consumed.
   crypto.v1alpha1.Nullifier nullifier = 1;
   // The fee allows `SwapClaim` without an additional `Spend`.
   crypto.v1alpha1.Fee fee = 2;
   // Note output for asset 1.
-  crypto.v1alpha1.EncryptedNote output_1 = 3;
+  crypto.v1alpha1.StateCommitment output_1_commitment = 3;
   // Note output for asset 2.
-  crypto.v1alpha1.EncryptedNote output_2 = 4;
+  crypto.v1alpha1.StateCommitment output_2_commitment = 4;
```

Comment:
The `EncryptedNote`s are replaced with bare `StateCommitment`s, because their contents are deterministically derived from the `SwapPlaintext`+`BatchSwapOutputData`.